### DISCUSSION TITLE
ようこそ n-playspace-http のディスカッションへ！

### DISCUSSION THREAD

**Discussion by satsukizzz** [satsukizzz/n-playspace-http#1] [Announcements]
## 👋 Welcome!
  私たちはコミュニティの他のメンバーとつながるための場所としてディスカッションを使っています。私たちは、あなたが以下のようなことを望んでいます。
  * 疑問に思ったことを質問する。
  * アイデアを共有してください。
  * 他のコミュニティメンバーと交流する。
  * 他者を歓迎し、オープンマインドであること。これは、私たちが一緒に作るコミュニティであることを忘れないでください。
  一緒に作っていくコミュニティであることを忘れないでください💪。

  まずは、下記のコメント欄に自己紹介と、このコミュニティで何をするかを教えてください。

www.DeepL.com/Translator（無料版）で翻訳しました。

以下、英語原版です。

## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by satsukizzz** [comment]
自己紹介！だって！

- 呼び方：さつき
- 好きなもの：飲み物！コーヒーやら紅茶やらジンジャエールやら。そして読書。音ゲー。
- 性格：教育好き、理詰め🤔
- プログラミングで何したい？：身の回りの雑事を「ボタン一発」で解決したい
- その他何でも！：末尾のzzzは眠い証拠です

ちょっとしたリポジトリながら、オーナーで皆さん引っ張っていこうと思います！よろしくお願いします！

**Comment by kenton116** [comment]
自己紹介
- 呼び方： kenton
- 好きなもの：プログラミング、工作。
- 性格：几帳面
- プログラミングで何したい？：自分のやりたいことを作る！
- その他：116は僕の大好きな番号ですw

**Comment by kenton116** [comment]
よろしくお願いします。

**Comment by fuuchin** [comment]
よろしくお願いします。  
自己紹介  

- 呼び方： ふーちん

- 好きなもの：プログラミング、slack、果物

- 性格：やりたくないことをダラダラする

- プログラミングで何したい？：先生みたいになりたい！言語とか作ってみたい！

- その他：眼鏡をかけてます

**Comment by Megafriday** [comment]
GitHubを使っての複数人でのコーディングをやった事が無いので、色々と勉強をさせて頂きたいと思い、参加させて頂きました。　既にこのプロジェクトには沢山の情報があり、大変勉強になっています。どうぞよろしくお願いいたします。

- 呼び方：めがふらいでー
- 好きなもの：プログラミング、スキューバダイビング、格闘技観戦
- 性格：N予備校の夏コンテストで優勝された方のサイトでは、アナコンタイプと診断されました。
- プログラミングで何したい？：ECサイト、自動売買、ゲーム制作

**Comment by nyeoro-bong** [comment]
みなさんこんにちは。”にょろぼん”と申します。
自己紹介さしあげます。よろしくおねがいします！

- 呼び方：　にょろぼん
- 好きなもの：　ドンレミーのプリンアラモード
- 性格：　ド文系。こつこつねばる秋田県人
- プログラミングで何したい？：『今、ここ、自分』に響く道具をささっと作る
- その他何でも！：　nianticさんに配慮してひらがなにしてますが、ポ○モンのアレです💦

**Comment by yuukimaru130516** [comment]
このディスカッションに気づかず、自己紹介していませんでした(^^;

- 呼び方： ゆうきまる
- 好きなもの： お寿司と音楽を聴くこと（特に柴咲コウさんが大好きです(^▽^)/）
- 性格： 几帳面です（笑）
- プログラミングで何したい？： なんでも自動化できるように頑張ってます
- その他何でも！： 現在、大学3年生のため就活中です。地味に忙しいため、あんまり活動できないかもしれないですがコツコツ
頑張りたいです！よろしくお願いします。ちなみに、大学では統計学を専攻しています！

**Comment by nrt0** [comment]
slackで「td」という名前です。
よろしくお願いします。

- 呼び方：てぃーでぃー
- 好きなもの：コーヒー、生き物、自然
- 性格：気になることがあったら調べるのが好き
- プログラミングで何したい？：仕事の効率化、ゲーム制作
- その他：色々学びたいです。

**Comment by HappyKumamoto** [comment]
リーダー&ここの仲間たち>貴重な体験をさせてくださりありがとうございます。

- 呼び方 : HappyK
- 好きなもの : 自転車、筋トレ
- 性格 : のんびり
- プログラミングで何したい？: 面白いものを作りたい
- その他 : Herokuに公開を区切りに離脱しますが、ずっと応援します！
